cmd/tendermint/commands/reset.go

```diff
@@ -42,21 +47,37 @@ var ResetPrivValidatorCmd = &cobra.Command{
 	Use:     "unsafe-reset-priv-validator",
 	Aliases: []string{"unsafe_reset_priv_validator"},
 	Short:   "(unsafe) Reset this node's validator to genesis state",
-	Run:     resetPrivValidator,
 	PreRun:  deprecateSnakeCase,
+	RunE:    resetPrivValidator,
 }
 
 // XXX: this is totally unsafe.
 // it's only suitable for testnets.
-func resetAllCmd(cmd *cobra.Command, args []string) error {
-	return resetAll(config.DBDir(), config.P2P.AddrBookFile(), config.PrivValidatorKeyFile(),
-		config.PrivValidatorStateFile(), logger)
+func resetAllCmd(cmd *cobra.Command, args []string) (err error) {
+	config, err = ParseConfig()
+	if err != nil {
+		return err
+	}
+
+	return resetAll(
+		config.DBDir(),
+		config.P2P.AddrBookFile(),
+		config.PrivValidatorKeyFile(),
+		config.PrivValidatorStateFile(),
+		logger,
+	)
 }
 
 // XXX: this is totally unsafe.
 // it's only suitable for testnets.
-func resetPrivValidator(cmd *cobra.Command, args []string) {
+func resetPrivValidator(cmd *cobra.Command, args []string) (err error) {
+	config, err = ParseConfig()
```

Comment:
I don't see `resetFilePV` relying on the global config value, so I'm not certain I see why we need to parse into the global var here.

Reply:
Huh? It literally calls `resetFilePV` with the values from `config` below.

Reply:
Yea, it uses the values from the variable as set here. I'm confused as to why we don't just use a local variable here for the return of `ParseConfig` instead of binding its result to the package-global `config` variable seeing as `resetFilePV` does not use fields from the global `config` variable as far as I can tell.